example/lib/device_details/device_details_bloc.dart

```diff
@@ -38,22 +38,23 @@ class DeviceDetailsBloc {
     _deviceController = BehaviorSubject<BleDevice>.seeded(device);
 
     _connectionStateController =
-        BehaviorSubject<PeripheralConnectionState>.seeded(device.isConnected
-            ? PeripheralConnectionState.connected
-            : PeripheralConnectionState.disconnected);
+        BehaviorSubject<PeripheralConnectionState>.seeded(
+            PeripheralConnectionState.disconnected);
```

Comment:
Is it no longer possible that the peripheral is connected already when entering this screen?

Reply:
I found that `device.isConnected` flag was always `false` and was never changing to `true`, so I removed it completely. My guess is that it was just copy-pasted from another project. We can consider handling already connected devices but it sounds like a new task to me.